ytmusicapi/parsers/search.py

```diff
@@ -50,8 +50,11 @@ def parse_search_result(data, search_result_types, result_type, category):
     default_offset = (not result_type or result_type == "album") * 2
     search_result = {'category': category}
     video_type = nav(data, PLAY_BUTTON + ['playNavigationEndpoint'] + NAVIGATION_VIDEO_TYPE, True)
-    if not result_type and video_type:
-        result_type = 'song' if video_type == 'MUSIC_VIDEO_TYPE_ATV' else 'video'
+    if not result_type:
+        if search_result["category"] == "Albums":
```

Comment:
Hi @polvallverdu , thanks for your contribution. However, this will not work without localization. Note that "category" is not used anymore else for code flow for this reason. Only result_type is localization independent (after delocalization in l.59)

Reply:
Ahh, I'll try to investigate it later today.